### DISCUSSION TITLE
Meeting record and summarize

### DISCUSSION THREAD

**Discussion by flefevre** [open-webui/open-webui#3378] [Ideas]
Hey there,
Is there any plugin that we could install in open WebUI just in order to record a meeting, to make a MP3, then to propose to make a stt with diarisation, and to propose to summarise the meeting?
Thanks

**Comment by justinh-rahb** [comment]
This could likely be done with Pipelines and maybe with the upcoming WebUI internal Functions/Filters/Tools.

**Comment by flefevre** [comment]
Do you have any advice on how to do it, so i can look a little on it?
Perhaps idea is to be able to upload a wav or mp3 file in order it is transcript?
